### DISCUSSION TITLE
How to handle versioning of GEMPAK?

### DISCUSSION THREAD

**Discussion by akrherz** [Unidata/gempak#74] [Q&A]
Historically, the Unidata GEMPAK release version has followed the NCEP NAWIPS release version verbatim.  When NCEP made a release, Unidata would follow with a release.

We may be in a situation now without further NCEP releases and/or the desire to make more frequent GEMPAK releases by the community / me :)  The question is how to handle the version number?

Presently, `gempak/source/gemlib/ss/ssvers.f` just has this:
```fortran
        CHARACTER*(*)   vmsg
C------------------------------------------------------------------------
        iret = 0
C
C*      Set the version string and add a null to the end.
C
        vmsg = 'Version 7.14.0'
        CALL ST_NULL ( vmsg, vmsg, lens, ier )
C*
        RETURN
        END
```

I can not find code that attempts to parse this value / do fancy things like `semver` checking.

My proposal is to add an additional release point to the version, so we have something like `Version 7.14.0.1`.  Anybody with better ideas / objections?

I believe going to `7.14.1` is a bad idea as it would conflict, if by chance NCEP made a `7.14.1` release.

**Comment by mzuranski** [comment]
I was just thinking about this too.  I agree with your `Version 7.14.0.1` idea, that makes more sense to me than anything else.

**Reply by ChaseStudebaker** [reply]
how do i download this

**Reply by akrherz** [reply]
> how do i download this

Source tarballs are available on the [github release page](https://github.com/Unidata/gempak/releases/tag/7.14.0.1)
